#!/usr/bin/env python
PACKAGE = "hanse_navigation"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("hysteresis_heading", double_t, 0, "in radians", 0.17, 0.1, 1)
gen.add("hysteresis_goal", double_t, 0, "", 2.0, 0.1, 4.0)
gen.add("forward_max_speed", double_t, 0, "", 0.8, 0.1, 1.0)
gen.add("forward_max_dist", double_t, 0, "", 4, 0.8, 4.0)
gen.add("angular_min_speed", double_t, 0, "", 0.3, 0.1, 1.0)
gen.add("angular_max_speed", double_t, 0, "", 0.6, 0.1, 1.0)
gen.add("p_heading", double_t, 0, "", 0.5, -10.0, 10.0)
gen.add("simulator", bool_t, 0, "", False)

exit(gen.generate(PACKAGE, "navigation", "Navigation"))
